extension-readme.md

```diff
@@ -1,182 +1,242 @@
+___
 
-## <a id="additional-setup-steps"></a>Additional Setup Steps
+# <a id="additional-setup-steps"></a>Setup
 
-After you install the AWS Toolkit for Visual Studio Code, you must complete these additional steps to access most of its features:
+After you install the AWS Toolkit for Visual Studio Code, to access most of its features, you must complete the additional steps defined in the [Getting Started](https://docs.aws.amazon.com/console/toolkit-for-vscode/getting-started) topic of the _AWS Toolkit for Visual Studio Code User Guide_. These additional steps include the following:
 
-1. [Create an AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) and an [access key](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/).
-1. Connect the Toolkit to AWS by using [AWS credentials](#managing-aws-resources).
+1. Create an AWS account (see the [Prerequisites](https://docs.aws.amazon.com/console/toolkit-for-vscode/setup-toolkit#setup-prereq) in the user guide and also these [additional details](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/))
+1. Create and configure a set of AWS credentials (see **Step 1** of [Managing AWS Resources](#managing-aws-resources))
+1. Connect the Toolkit to AWS using those credentials (see **Step 2** of [Managing AWS Resources](#managing-aws-resources))
 
 To use this Toolkit to develop [serverless applications with AWS](https://aws.amazon.com/serverless/), you must also do the following on the local machine where the Toolkit is installed:
 
-1. Install the [AWS Command Line Interface (AWS CLI)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
-1. Install and then run [Docker](https://docs.docker.com/install/).
-1. Install the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html).
+1. Install the AWS Command Line Interface (AWS CLI)
+1. Install and start Docker (also see this general information about [Docker](https://docs.docker.com/install/))
+1. Install the AWS SAM CLI
 
-For complete setup instructions, see [Setting Up the AWS Toolkit for Visual Studio Code](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/getting-started.html) in the _AWS Toolkit for Visual Studio Code User Guide_.
+See [Installing the AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) for complete setup instructions for these three components.
 
-[Return to Top](#top)
+{ [Return to Contents](#contents) } or { [Return to Top](#top) }
+___
 
-## <a id="usage"></a>Usage
+# <a id="aws-commands"></a>AWS Commands
+
+The AWS Toolkit for Visual Studio Code has several features that you can access through the [Command Palette](#open-command-palette) (select **View**, then **Command Palette**):
+
+| AWS Command | Description |
```

Comment:
This list needs to be updated.

Reply:
Do we want to finish these current changes and then update the list later, or wait until it's appropriate to update the list to finish all the changes at once?

Reply:
I think it's appropriate to update the list now considering that the commands are currently in the toolkit. @awschristou , thoughts?

Reply:
Done.